Dockerfile.pipelines

```
@@ -78,11 +78,16 @@ RUN if [ "$MPI_INSTALL" == "1" ] ; then \
     fi
 
 # Set up conda
-RUN wget --no-verbose https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O /tmp/miniconda.sh
-RUN bash /tmp/miniconda.sh -b -p /usr/local/miniconda
+RUN if [ $installWithPip == "true" ] ; then \ 
+      apt-get install -y python3.6-dev python3-pip ; \
+    else \
+      wget --no-verbose https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh ; \
+      bash /tmp/miniconda.sh -b -p /usr/local/miniconda ; \
```

```suggestion
      bash /tmp/miniconda.sh -b -p /usr/local/miniconda ; \
      export PATH=/usr/local/miniconda/bin:$PATH
```
We need an extra bit of path setting here so the following conda commands work. It should be safe to leave the ENV command in line 90 as if there's no conda installed it won't have any effect.

---

Yeah these lines are giving me trouble so far no luck having it to run. Like in travis I'm just trying to skip the conda installation to make sure we don't get lucky with pip getting so extras from conda.